ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/ScanResultsOverviewTable.tsx

```diff
@@ -0,0 +1,100 @@
+const defaultSortOption = { field: 'Last Scanned', direction: 'asc' } as const;
+
+function ScanResultsOverviewTable() {
+    const { page, perPage, onSetPage, onPerPageSelect } = usePagination();
+    const { sortOption, getSortParams } = useTableSort({ sortFields, defaultSortOption });
+    const [searchFilter, setSearchFilter] = useState<SearchFilter>(defaultSearchFilter);
+    const [scanResultsOverviewData] = useState<ComplianceScanResultsOverview[]>(
+        scanResultsOverviewMockData.scanOverviews
+    );
+
+    useEffect(() => {
+        // TODO: fetch GetComplianceScanResultsOverview once implemented and update scanResultsOverviewData
+    }, [searchFilter, page, perPage, sortOption]);
+
+    const defaultActions = (): IAction[] => [
+        {
+            title: 'Edit schedule',
+            // TODO: link to edit schedule page
+        },
+    ];
+
+    const displayOnlyItemOrItemCount = (items: string[], multipleItemLabel: string): string => {
+        if (items.length > 1) {
+            return `${items.length} ${multipleItemLabel}`;
+        }
+        return items[0];
+    };
+
+    return (
+        <>
+            <ScanResultsToolbar
+                numberOfScanResults={scanResultsOverviewData.length}
+                searchFilter={searchFilter}
+                setSearchFilter={setSearchFilter}
+                page={page}
+                perPage={perPage}
+                onSetPage={onSetPage}
+                onPerPageSelect={onPerPageSelect}
+            />
+            <TableComposable borders={false}>
+                <Thead noWrap>
+                    <Tr>
+                        <Th sort={getSortParams('Scan Name')}>Scan</Th>
+                        <Th>Clusters</Th>
+                        <Th>Profiles</Th>
+                        <Th sort={getSortParams('Failing Controls')}>Failing Controls</Th>
+                        <Th sort={getSortParams('Last Scanned')}>Last Scanned</Th>
+                        <Td />
+                    </Tr>
+                </Thead>
+                <Tbody>
+                    {scanResultsOverviewData.map(({ scanStats, clusterId, profileName }) => {
+                        return (
+                            <Tr key={scanStats.scanName}>
+                                <Td>{scanStats.scanName}</Td> {/* TODO: link to scan details  */}
+                                <Td>{displayOnlyItemOrItemCount(clusterId, 'clusters')}</Td>
+                                <Td>{displayOnlyItemOrItemCount(profileName, 'profiles')}</Td>
+                                <Td>{`${scanStats.numberOfFailingChecks}/${scanStats.numberOfChecks}`}</Td>
+                                <Td>{format(scanStats.lastScan, 'DD MMM YYYY, h:mm:ss A')}</Td>
+                                <Td isActionCell>
+                                    <ActionsColumn items={defaultActions()} />
+                                </Td>
+                            </Tr>
```

Comment:
Brad, can you double check future plans to see whether actions consist of either READ_WRITE or READ_WRITE_ACCESS or all READ_WRITE_ACCESS? If it looks like write, a good follow up is conditional rendering for the table head and data cells.

For example,
1. https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/VulnReportsPage.tsx#L70-L74
2. https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/VulnReportsPage.tsx#L295
3. https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/VulnReportsPage.tsx#L494-L501

Reply:
Thats a good callout, I'll make a note and bring it up in our next sync